profiler/upload.go

```diff
@@ -16,6 +16,8 @@ import (
 	"net/http"
 	"strings"
 	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
```

Comment:
What happens if we release a major version bump?

Reply:
The next major version will be either `gopkg.in/DataDog/dd-trace-go.v2/profiler/...` or likely `github.com/DataDog/dd-trace-go/v2/profiler/...` if we opt into modules (which we likely will).